brave/brave-ios
# Fix #4335: Deprecate ours HTTPSE in favor of the one bundled in WKWeb…

…View.



## Summary of Changes

Security review https://github.com/brave/security/issues/640


This pull request fixes #4335 


## Submitter Checklist:


## Test Plan:

Note for testers: this feature has been enabled on iOS 15 devices by default, we kept using Apple's and our implementation for some time.

This is a simple flag we turn on, we can't decide what websites are upgraded or not, we do not also know what websites we can test this on. I found apache.org to be a good candidate for testing.

Use iOS 15 device.
Enabled case:
- Go to http://apache.org or enter `apache.org` in url bar
- Verify that the website upgraded itself to https

Disabled case:
- Go to Brave settings -> Shields and disable HTTPS upgrades
- Go to http://apache.org or enter `apache.org` in url bar
- Verify that the website did not upgrade to https

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
